dashboard_2/articles/tests/test_article_model.py

```diff
@@ -45,6 +45,16 @@ def test_can_get_version_data_for_multi_version_article(article,
 	assert data == article_mutli_version_response_data
 
 
+@pytest.mark.django_db
+def test_will_not_add_property_if_version_value_is_zero(article):
```

Comment:
It would be helpful to have a diagram of the new data model just so we can understand what is going to substitute those changes (a before and after would be even better). I checked with @jhroot that version 0 was used to hold properties common to all versions. It will be good to have a picture so we can reason your plan to treat those cases. Thanks.

Reply:
The data model itself has not changed at all. I intentionally kept it the same for this piece of work so not to break any existing functionality. 

The test case in question is only to do with evaluating article versions, meaning when you ask for all versions of an article you don't get a 'zero' version entry in the returned data.

I saw the existing use of the zero version events and have kept it the same.

I will, however, update this test case name as in hindsight it is a little misleading without additional context.